docs/NSR.md

```diff
@@ -64,16 +64,10 @@ The following are outcomes that this document seeks to achieve:
 
 **CA Infrastructure**: Collectively the infrastructure used by the CA or Delegated Third Party which qualifies as a:
 
-* Certificate Management System;
-* Certificate System;
-* Delegated Third Party System;  
-* Issuing System;
-* Root CA System (Air-Gapped and otherwise); or
-* Security Support System.
+* Certificate System; or
+* Root CA System (Air-Gapped and otherwise).
 
-**Certificate Management System**: A system used by a CA or Delegated Third Party to process, approve issuance of, or store certificates or certificate status information, including the database, database server, and storage.
-
-**Certificate System**: A system used by a CA or Delegated Third Party to access, process, or manage data or provide services related to:
+**Certificate System**: A system used by a CA or Delegated Third Party to access, process, or manage data or provide services related to performing:
```

Comment:
> to store, access, process, or manage data

Reply:
Cloud Services Subgroup Concern: Removal of Support Systems from Certificate Systems no longer encompasses those periphery systems in any kind of requirements for controls. How can we incorporate those requirements into this definition?